frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/FrontendPluginFactory.java

```diff
@@ -3,34 +3,45 @@
 import java.io.File;
 
 public final class FrontendPluginFactory {
-    
+
```

Comment:
It's not necessary to fix all code style things here, adding new methods is enough. (I suppose code style fixes could be a separate PR)